models/tridimensional/kleinstiver_csv_to_dict.py

```diff
@@ -0,0 +1,65 @@
+"""
+Convert CSV of Cas9 mutants from Klienstiver et al. (2015), Nature, doi:10.1038/nature14592 into dict format
+Included for reproducibility of data preparation
+"""
+
+from cas9_mutants import PI_domain, PI_sec_structure, aa_group
+import os
+import csv
+
```

Comment:
minor stuff on import order convention: usually all external libs are imported first, alphabetically (the ones starting with import ..., then the ones starting with from ... import blah, then the local libs are imported the same way

pep guide: Imports should be grouped in the following order:
standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.

`import csv`
`import os`

`from cas9_mutants import PI_domain, PI_sec_structure, aa_group`

I also think its more common to put the imports at the very top, even top-level comments